### PR TITLE
Bigtop 4238: New metrics added to the Agent and the Server retrieves metrics and provides an interface

### PR DESCRIPTION
**Pull request by Cat-Drink**

### CODE REVIEW

File: bigtop-manager-agent/src/main/java/org/apache/bigtop/manager/agent/monitoring/AgentHostMonitoring.java [hunk 1]
```diff
@@ -336,7 +421,10 @@ public static void diskMultiGaugeUpdateData(MultiGauge diskMultiGauge) {
 
     public static void memMultiGaugeUpdateData(MultiGauge memMultiGauge) {
         try {
+            // getHostInfo 获取全部数据
+            // getMEMGauge 解析内存数据
```
Comment (wuchunfu) [inline bigtop-manager-agent/src/main/java/org/apache/bigtop/manager/agent/monitoring/AgentHostMonitoring.java:425]: @Cat-Drink Please revise the comments to English, thanks

File: bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/controller/MonitoringController.java [hunk 2]
```diff
@@ -42,6 +42,21 @@ public class MonitoringController {
     @Operation(summary = "agent healthy", description = "agent healthy check")
     @GetMapping("agenthealthy")
     public ResponseEntity<JsonNode> agentHostsHealthyStatus() {
+        // 以json格式返回   响应结果 参数即为数据  数据格式为 json
```
Comment (wuchunfu) [inline bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/controller/MonitoringController.java:45]: Same modification to English

**Comment by kevinw66** [comment]
@Cat-Drink You don't need to close the PR, just push your new commit to this branch and the change will be synchronized.

**Comment by Cat-Drink** [comment]
> @Cat-Drink You don't need to close the PR, just push your new commit to this branch and the change will be synchronized.

ok , thanks, i have known, now you can check this latest PR.

**Pull request closed without merging**